doc/faq.rst

```diff
@@ -244,3 +243,24 @@ program: Insert the following instructions in your main script::
 
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
+
+How do I set a random_state for an entire execution?
+----------------------------------------------------
+
+For testing and replicability, it is often important to have the entire execution
+controlled by a single seed for the pseudo-random number generator used in algorithms
+that have a randomized component. Scikit-learn does not use its own global random state;
+whenever a RandomState instance or an integer random seed is not provided as an argument,
+it relies on the numpy global random state, which can be set using ''numpy.random.seed'' function.
```

Comment:
ah got it. I was using github markdown and adding two " ' "s ; having only one did the trick. Thanks!